lib/sonos.js

```diff
@@ -1171,6 +1174,117 @@ Sonos.prototype.getQueue = function (callback) {
   })
 }
 
+// Make Sonos an EventEmitter
+util.inherits(Sonos, EventEmitter)
```

Comment:
I think this line breaks node v4, is there a reason why you would event test against this 2 year old version?

Reply:
I'd be happy to drop support for node v4 now. We're now into just maintenance on the [official support](https://github.com/nodejs/Release#release-schedule), there's no need for us to bring new functionality to a node version going out the window.